whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/AttributeSyntax.java

```diff
@@ -898,11 +901,12 @@ public boolean matches(final ObjectType objectType, final String value) {
         public String getDescription(final ObjectType objectType) {
             return "" +
                     "It should contain 2 to 10 words.\n" +
-                    "Each word consists of letters, digits or the following symbols:\n" +
+                    "A word is made up of ASCII characters and additionally:\n" +
```

Comment:
be more precise, `A word is made up of ASCII alphanumeric characters....`, as not all `ASCII characters` are allowed.